hs_core/templates/email/group_membership.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+{% block main %}
+    {% if group_owner %}
+        <p>{% trans "Dear " %}{{ group_owner.first_name }}</p>
+        <p>{% trans "User " %}{{ user.first_name }} has requested to join your HydroShare group ({{ group.name }}). Please click on the link below to accept this request.</p>
```

Comment:
@pkdash Should the full name of the requesting user, perhaps along with organization info, be shown here in the request email to the group owners for accepting or rejecting requests so that the group owner knows a bit of more info to consider whether to accept user's request?